tests/test_cmd_test.py

```diff
@@ -89,7 +89,7 @@ def test_data_manager_docker_mount(self):
     @skip_if_environ("PLANEMO_SKIP_GALAXY_TESTS")
     def test_workflow_test_simple_yaml(self):
         """Test testing a simple YAML workflow with Galaxy."""
-        with self._isolate():
+        with self._isolate() as test_dir:
             random_lines = os.path.join(PROJECT_TEMPLATES_DIR, "demo", "randomlines.xml")
```

Comment:
Not related, but why is the `random_lines1` tool added as `--extra_tools` below (and in another test), even if the tool is not used in the `wf1.gxwf.yml` workflow?

Reply:
maybe to check that multiple `--extra_tools` work ?